packages/mobile/src/components/details-tile/DetailsTile.tsx

```diff
@@ -220,6 +222,8 @@ export const DetailsTile = ({
   const navigation = useNavigation()
 
   const currentUserId = useSelector(accountSelectors.getUserId)
+  const currentPlayerTrackId = useSelector(getTrackId)
+  const isCurrentTrack = track && track.track_id === currentPlayerTrackId
```

Comment:
Can you put this equality check into the selector, that will prevent extra rerenders of the details tile when the playing track changes:

```suggestion
  const isCurrentTrack = useSelector((state) => {
    return track && track.track_id === getTrackId(state)
 }
```

Reply:
Also same deal on web

Reply:
🧠